### DISCUSSION TITLE
Promote newsletter

### DISCUSSION THREAD

**Discussion by odysseuskir** [simplified-coding/simplified-coding#152] [Ιδέες / Σχόλια]
Our newsletter currently has 0 subscribers. I propose we promote the newsletter on the home page, in order to get more traffic as well as our Discord server. Furthermore, we can move away from Google Forms, by having a Tally form for users to enter their email in.

**Comment by GalaxyGamingBoy** [comment]
A tally form already exists

**Reply by odysseuskir** [reply]
Good. We can look on implementing it inside the website later on as part of the newsletter promotion.
